client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -98,10 +99,12 @@ class RequiredPluginsInstallView extends Component {
 		}
 
 		if ( waitingForPluginListFromSite ) {
-			this.setState( {
-				message: translate( 'Waiting for plugin list from site' ),
-				progress: 0,
-			} );
+			if ( workingOn !== 'WAITING_FOR_PLUGIN_LIST_FROM_SITE' ) {
+				this.setState( {
+					message: translate( 'Waiting for plugin list from site' ),
```

Comment:
FYI - we soon won't have messages anymore ( https://github.com/Automattic/wp-calypso/pull/17317 )